sync/rpc/server.go

```diff
@@ -209,9 +228,11 @@ func (s *SyncAgentServer) BackupCreate(ctx context.Context, req *BackupCreateReq
 	reply := &BackupCreateReply{
 		Backup: backupID,
 	}
-	fmt.Fprintf(os.Stdout, reply.Backup)
 
-	s.backupList.Store(backupID, replicaObj)
+	s.backup.BackupCreate(backupID, replicaObj)
+	if err := s.backup.BackupUpdate(); err != nil {
+		logrus.Errorf("failed to cleanup old backups:%v", err)
```

Comment:
Space after `:`.

Reply:
This line is removed as refresh doesn't return error